src/engine/Sort.cpp

```diff
@@ -76,7 +76,7 @@ ResultTable Sort::computeResult() {
   LOG(DEBUG) << "Sort result computation..." << endl;
   ad_utility::Timer t{ad_utility::timer::Timer::InitialStatus::Started};
   IdTable idTable = subRes->idTable().clone();
-  runtimeInfo().addDetail("time-cloning", t.msecs());
+  runtimeInfo().addDetail("time-cloning", t.msecs().count());
```

Comment:
I think it would be really nice if `addDetail` had a 
overload for `milliseconds` as the second argument. We will add times to the details more often.